docker/Dockerfile

```diff
@@ -17,5 +17,4 @@ WORKDIR /go/src/github.com/muesli/beehive
 VOLUME /conf
 
 # This form of ENTRYPOINT allows the beehive process to catch signals from the `docker stop` command
-ENTRYPOINT ["beehive", "-config", "/conf/beehive.conf", "-bind", "0.0.0.0:8181", "-canonicalurl", ${CANONICAL_URL}]
-
+ENTRYPOINT beehive -config /conf/beehive.conf -bind 0.0.0.0:8181 -canonicalurl ${CANONICAL_URL}
```

Comment:
I'm worried that changing this form invalidates what the comment above states. Any insight on this?

Reply:
`docker stop` will always shutdown the container, changing `ENTRYPOINT` will not affect the container to catch stop signal.

Reply:
LGTM.
But `ENTRYPOINT ["beehive", "-config", "/conf/beehive.conf", "-bind", "0.0.0.0:8181", "-canonicalurl", ${CANONICAL_URL}]` works well for me though it did not been tested in many cases.